src/plugins/data_view_field_editor/public/components/field_format_editor/editors/color/color.tsx

```diff
@@ -195,7 +195,7 @@ export class ColorFormatEditor extends DefaultFormatEditor<ColorFormatEditorForm
         render: (item: IndexedColor) => {
           return (
             <div
-              style={{
+              css={{
```

Comment:
Revert this since styles are dynamic
```suggestion
              style={{
```